react-graphql-apollo-tutorial.md

```diff
@@ -456,22 +456,22 @@ const client = new ApolloClient({
 });
 {{< /highlight >}}
 
-In order to initialize Apollo Client, you must specify link and cache properties on the config object. Once you start your application again, everything should run without any errors. If it doesn't check whether you have implemented a basic App component in your *src/App/index.js* file. After all, the `ReactDOM` API needs to hook this component into the HTML.
+To initialize Apollo Client, you must specify link and cache properties on the config object. Once you start your application again, there should be no errors. It doesn't check whether you have implemented a basic App component in your *src/App/index.js* file because the `ReactDOM` API needs to hook this component into the HTML.
```

Comment:
My wording was bad here. It should have said: "If it doesn't, check whether you have implemented a basic App component in your *src/App/index.js* file because the `ReactDOM` API needs to hook this component into the HTML."